plugins/nginx-vhosts/templates/nginx.ssl.conf.template

```diff
@@ -6,6 +6,21 @@ server {
 }
 
 server {
+  listen      [::]:80;
+  listen      80;
+  server_name $SSL_SERVER_NAME;
+  return 301 https://$SSL_SERVER_NAME$request_uri;
+}
+
+server {
+  listen      [::]:80;
+  listen      80;
+  server_name www.$SSL_SERVER_NAME;
+  return 301 https://www.$SSL_SERVER_NAME$request_uri;
```

Comment:
Adding this would maybe break what worked (or didn't) before, so I would remove these for now and we can revisit during a major release.

A better case would be to document the redirect situation so people know what works/doesn't and how to change it :)